### PR TITLE
Add test cases

### PR DESCRIPTION
**Pull request by tzihiang**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/cookbook/CookbookSearchByInventoryCommandTest.java [hunk 1]
```diff
@@ -23,4 +24,15 @@ public void execute_validInput() {
     public void execute_null_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> new CookbookSearchByInventoryCommand().execute(null));
     }
+
+    @Test
+    public void execute_success() {
+
+    }
```
Comment (teo-jun-xiong) [inline src/test/java/seedu/address/logic/commands/cookbook/CookbookSearchByInventoryCommandTest.java:31]: this is not required, it's the same as `execute_validInput()`

File: src/test/java/seedu/address/testutil/RecipeBuilder.java [hunk 2]
```diff
@@ -70,7 +72,26 @@ public RecipeBuilder withTags(String ... tags) {
...
+     * Adds the {@code ingredient} to the ingredient list of the recipe.
+     * @param ingredient
+     */
+    public RecipeBuilder withIngredients(Ingredient ingredient) {
+        this.ingredients.add(ingredient);
+        return this;
+    }
+
+    /**
+     * Adds the {@code steps} to the steps list of the recipe.
```
Comment (teo-jun-xiong) [inline src/test/java/seedu/address/testutil/RecipeBuilder.java:85]: `{@code ...}` must refer to the class, so: `{@code Step}`

File: src/test/java/seedu/address/testutil/RecipeBuilder.java [hunk 3]
```diff
@@ -70,7 +72,26 @@ public RecipeBuilder withTags(String ... tags) {
         return this;
     }
 
+    /**
+     * Adds the {@code ingredient} to the ingredient list of the recipe.
```
Comment (teo-jun-xiong) [inline src/test/java/seedu/address/testutil/RecipeBuilder.java:76]: `{@code ...}` must refer to the class, so: `{@code Ingredient}`

**Review by teo-jun-xiong — APPROVED** [review]

**Comment by tzihiang** [comment]
Updated the ModelManager, ModelManagerTest java. Thanks for looking out!

**Review by hans555 — APPROVED** [review]

**Review by shanecsj — APPROVED** [review]

**Pull request merged**